ipv-api/src/main/java/uk/gov/di/authentication/ipv/entity/IdentityProgressResponse.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.di.authentication.ipv.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import com.nimbusds.oauth2.sdk.id.State;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.net.URI;
+
+public class IdentityProgressResponse {
+
+    @SerializedName("status")
+    @Expose
+    @Required
+    private IdentityProgressStatus status;
+
+    @SerializedName("client-name")
```

Comment:
There doesn't seem to be a consistent casing for serialized name. I've chosen the same casing as AuthCodeHandler, as that's the other orchestration handler we'll have